bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -25,7 +25,7 @@ limitations under the License.
   <artifactId>bigtable-beam-import</artifactId>
 
   <properties>
-    <mainClass>com.google.cloud.bigtable.beam.sequencefiles.Main</mainClass>
+    <mainClass>com.google.cloud.bigtable.beam.Main</mainClass>
```

Comment:
what are our current instructions to users on how to use this? Will this break existing implementations?

Reply:
This class path is only specified here, so it should be transparent to the outside users.

Reply:
Yes makes sense, my larger question is do we expect users to already be importing `com.google.cloud.bigtable.beam.sequencefiles.Main` in source files that would break them by moving its package?

Reply:
I since that is unlikely but should be fine as well since we left the original file in the old location, too.

Reply:
Oooh ok perfect. Thanks!